src/pages/AddRestaurantPage1/AddRestaurantPage1.jsx

```diff
@@ -36,42 +92,34 @@ function AddRestaurantPage1() {
         return;
       }
 
+      if (!restaurantName) {
+        alert("Please select a restaurant name");
+        return;
+      }
+  
       const Post = Parse.Object.extend("Post");
       const post = new Post();
       post.set("userId", currentUser.id);
       post.set("restaurantName", restaurantName);
       post.set("restaurantAddress", restaurantAddress);
       post.set("savedToList", selectedList);
       post.set("text", restaurantComment);
-
-      if (selectedImage) {
-        const base64Image = await convertImageToBase64(selectedImage);
-        const file = new Parse.File("restaurantImage.png", {
-          base64: base64Image,
-        });
+  
+      if (selectedImage instanceof File) {
+        const file = new Parse.File("image.jpg", selectedImage);
```

Comment:
does it still work even if image is a .png or something else?

Reply:
For png yes! Not sure about others but maybe limiting to jpg and png is good